packages/polling-controller/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0]
+### Added
+- Add `PollingControllerOnly` to extend from an empty class. This will allow classes that previously are just classes that don't extend from BaseV1 or V2 to extend from this new `PollingControllerOnly`. ([#1873](https://github.com/MetaMask/core/pull/1873))
+
+### Changed
+- **BREAKING:** `_executePoll()` is called immediately on start if no polling interval is already active for the networkClientId + options combination ([#1874](https://github.com/MetaMask/core/pull/1874))
+- Bump `@metamask/auto-changelog` from 3.2.0 to 3.4.0 ([#1870](https://github.com/MetaMask/core/pull/1870))
```

Comment:
This is a development dependency, so we can drop this.

Reply:
done https://github.com/MetaMask/core/pull/1904/commits/9c4182af6cff40d18fce89480ef7cb339f4b9a3e